### PR TITLE
add VPN IPs to prompt

### PR DESCRIPTION
**Pull request by golgoth31**
add function to get vpn ips

### CODE REVIEW

**Comment by shibumi** [comment]
I like the idea of VPN prompts, but isn't there a better way as using `nmcli`? I know enough people that don't use `nmcli`.

**Comment by onaforeignshore** [comment]
`nmcli` is also linux specific, so no support on macOS

**Comment by onaforeignshore** [comment]
On macOS, VPN IP address can be found using: `ifconfig ppp0 | grep "inet " | grep -v 127.0.0.1 | awk '{print $2}'`. If VPN is inactive the result is `ifconfig: interface ppp0 does not exist`

**Comment by golgoth31** [comment]
Hi, thank you
I don't have any macOS so I can't test it. I will try to improve it through ifconfig or ip command

**Comment by golgoth31** [comment]
Done using "ip" command and no more nmcli.

File: powerlevel9k.zsh-theme [hunk 1]
```diff
@@ -1062,6 +1062,18 @@ prompt_symfony2_version() {
   fi
 }
 
+set_default POWERLEVEL9K_VPN_IP_INTERFACE "tun"
+# prompt if vpn active
+prompt_vpn_ip() {
+  if [[ -x /usr/bin/nmcli ]]; then
+    for vpn_iface in $(/bin/ip tuntap | grep -e ^"$POWERLEVEL9K_VPN_IP_INTERFACE" | cut -d":" -f1)
```
Comment (shibumi) [inline powerlevel9k.zsh-theme]: Don't use the absolut path here. The path to `ip` can change, depending on OS and distribution.

**Comment by golgoth31** [comment]
Ok, I removed the full path for the "ip" command

**Comment by shibumi** [comment]
Ok thanks, one more thing.. can you use the `next` branch instead of `master`? We work on the `next` branch for future releases and it would be also nice to have the commits `rebased` to one commit via `git rebase -i ...` :)

**Comment by golgoth31** [comment]
see pull #539

**Pull request closed without merging**